scripts/pi/wifi_scan.sh

```diff
@@ -0,0 +1,26 @@
+sudo iwlist wlan0 scanning | awk 'BEGIN{ FS="[:=]"; OFS = " " }
```

Comment:
iwlist and iw both have a huge output, iwlist was the easiest I to parse I felt.

example output from my device

```
./scripts/pi/wifi_scan.sh
60:E3:27:B8:D9:E4 "Pasaana" 2.462 11 -50 on
B8:27:EB:C1:2C:A9 "picobrew-logger" 2.412 1 -72 on
F0:72:EA:3B:7D:9A "Mortis" 2.462 11 -74 on
DC:A6:32:A4:39:DB "PICOBREW" 2.462 11 -47 on
7C:D9:5C:9E:DA:AA "Mortis" 2.462 11 -93 on
```